src/main/webapp/app/entities/quiz/short-answer-question.model.ts

```diff
@@ -7,6 +7,9 @@ export class ShortAnswerQuestion extends QuizQuestion {
     public spots?: ShortAnswerSpot[];
     public solutions?: ShortAnswerSolution[];
     public correctMappings?: ShortAnswerMapping[];
+    // default client values
```

Comment:
I think you can remove this comment because we know that they are default values